src/ert/scheduler/scheduler.py

```diff
@@ -181,32 +181,52 @@ async def execute(
         # We need to store the loop due to when calling
         # cancel jobs from another thread
         self._loop = asyncio.get_running_loop()
-        async with background_tasks() as cancel_when_execute_is_done:
-            cancel_when_execute_is_done(self._publisher())
-            cancel_when_execute_is_done(self._process_event_queue())
-            cancel_when_execute_is_done(self.driver.poll())
-            if min_required_realizations > 0:
-                cancel_when_execute_is_done(
+        long_lived_tasks = [
```

Comment:
We should call this something more precise, as these tasks are roughly just as longlived as the other ( = realization) tasks.

Reply:
"event_handler_tasks" ?

Reply:
hmm, that is maybe not a good name given we also add stop_long_running_jobs to the list.

Reply:
`long_lived_tasks` is semantically correct, but it does not say too much - I agree. `scheduler_maintenance_tasks`?

Reply:
What about just scheduler_tasks? The other tasks are job_tasks, which are dependent on the scheduler through the scheduler_tasks.

Reply:
`ensemble_tasks`?